ffi/passmanagers.cpp

```diff
@@ -244,10 +254,12 @@ LLVMPY_AddAlwaysInlinerPass(LLVMPassManagerRef PM, bool insertLifetime) {
     unwrap(PM)->add(llvm::createAlwaysInlinerLegacyPass(insertLifetime));
 }
 
+#if LLVM_VERSION_MAJOR < 15
 API_EXPORT(void)
 LLVMPY_AddArgPromotionPass(LLVMPassManagerRef PM, unsigned int maxElements) {
```

Comment:
Ditto with `ArgumentPromotionPass` in NPM as below

Reply:
Did you mean legacy version of this pass is available as well? I couldn't find it.